classes/class.tapestry.php

```diff
@@ -322,6 +323,40 @@ public function getAllContributors()
         return array_unique($authors, SORT_REGULAR);
     }
 
+    /**
+     * Retrieve a Tapestry post for export.
+     *
+     * @return object $tapestry
+     */
+    public function export()
+    {
+        $nodes = [];
+        foreach ($this->nodes as $node) {
+            $temp = (new TapestryNode($this->postId, $node))->get();
+            if (NodeStatus::DRAFT == $temp->status) {
+                continue;
+            }
+            $nodes[] = $temp;
```

Comment:
@wynnset @jaskwong How should we be handling submitted, accepted, and rejected nodes? Do those export too?

Reply:
DRAFT nodes should not be included (meaning submitted or rejected nodes). Accepted nodes should be included since they are not DRAFT.